actions/auth/auth.tsx

```diff
@@ -20,6 +21,11 @@ output = {
 }
 
 print(json.dumps(output), end="")
+
+---
+
+!metadata:getCreds:requirements.txt
```

Comment:
This is interesting, why do you need this?

Reply:
without this the python runtime for the gateway-creds tool doesn't get downloaded

Reply:
oh, is this because of how we are calling `evaluate` on it? normally i would expect those deps to be pulled automatically.

Reply:
I would as well, even with evaluate. Not sure if it's a bug or not, but this was required to make it work